src/inmanta/app.py

```diff
@@ -451,9 +451,10 @@ def export_parser_config(parser: argparse.ArgumentParser, parent_parsers: abc.Se
     parser.add_argument(
         "--delete-resource-set",
         dest="delete_resource_set",
-        help="Remove a resource set as part of a partial compile. This option can be provided multiple times and should always "
-        "be used together with the --partial option.",
-        action="append",
+        help="Remove resource set(s) passed in the INMANTA_REMOVED_SET_ID env variable as part of a partial compile. "
+        "This option should always be used together with the --partial option",
+        action="store_true",
```

Comment:
I think 
1. we should always handle the env var if it is set
2. I think we can allow both the option and the env var to be set at the same time, this makes the backward compatibility trivial.